doc/api/deprecations.md

```diff
@@ -2542,6 +2542,41 @@ accordingly instead to avoid the ambigiuty.
 To maintain existing behaviour `response.finished` should be replaced with
 `response.writableEnded`.
 
+<a id="DEP0144"></a>
+### DEP0144: `module.parent`
+<!-- YAML
+changes:
+  - version:
+    - v14.6.0
+    - REPLACEME
```

Comment:
I think this line should go above the previous since ascending version numbers are what are used in the rest of this document.

Reply:
Oops you're right